pom.xml

```diff
@@ -0,0 +1,110 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.insightsystems.symphony.dal.serverlink</groupId>
+    <artifactId>SLPSB1008H</artifactId>
+    <version>1.0.7.1</version>
+
+	<properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    </properties>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <version>3.1</version>
+                <configuration>
+                    <source>8</source>
+                    <target>8</target>
+                </configuration>
+            </plugin>
+        </plugins>
+    </build>
+
+    <dependencies>
+        <dependency>
+            <groupId>com.avispl.symphony.api</groupId>
+            <artifactId>symphony-api-commons</artifactId>
+            <version>5.0.0</version>
+            <scope>system</scope>
+            <systemPath>${basedir}/lib/symphony-api-commons-5.0.0.jar</systemPath>
+        </dependency>
+        <dependency>
+            <groupId>com.avispl.symphony.api</groupId>
+            <artifactId>symphony-api-dal</artifactId>
+            <version>5.0.0</version>
+            <scope>system</scope>
+            <systemPath>${basedir}/lib/symphony-api-dal-5.0.0.jar</systemPath>
+        </dependency>
+        <dependency>
+            <groupId>com.avispl.symphony.dal</groupId>
+            <artifactId>symphony-dal-commons</artifactId>
+            <version>5.0.0</version>
+            <scope>system</scope>
+            <systemPath>${basedir}/lib/symphony-dal-commons-5.0.0.jar</systemPath>
+        </dependency>
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-core</artifactId>
+            <version>2.9.9</version>
```

Comment:
can you please add <scope>provided</scope> for all dependencies that exist within Symphony dependencies